geps/gep-1709.md

```diff
@@ -37,8 +45,303 @@ which implementations can prove they satisfy and be recognized for.
 
 ## API
 
-TODO
+The API for conformance profiles will be a pipeline that implementations can
+opt into. The workflow is effectively:
+
+1. select a [profile](#profiles)
+2. [integrate](#integration) tests in the downstream project
+3. [report results and get certified](#certification)
+
+The goal is to make selecting a conformance profile as simple of a process as
+feasible and support both the existing command line integration approach (e.g. `go test`) as well
+as a [Golang][go] approach using the conformance suite as a library.
+
+[go]:https://go.dev
+
+### Profiles
+
+Initially there will be three profiles:
+
+- `Layer4`
+- `Layer7`
+- `Mesh`
+
+> **NOTE**: these are simply the initial profiles we're going to start with,
+> it's plausible for there to be more in the future.
+
+These named profiles are effectively categories which represent the high level
+grouping of tests. When conformance is reported using one of these profiles
+extra features can be covered according to support levels:
+
+- `extended`
+- `implementation-specific`
+
+> **NOTE**: `implementation-specific` doesn't really have much in the way of
+> tests today, but it is something users want to be able to display so it's
+> considered for reporting purposes.
+
+The technical implementation of these profiles is very simple: effectively a
+"profile" is a static compilation of existing [SupportedFeatures][feat] which
+represent the named category. Features that aren't covered under a "core" level
+of support are opt-in.
+
+[feat]:https://github.com/kubernetes-sigs/gateway-api/blob/c61097edaa3b1fad29721e787fee4b02c35e3103/conformance/utils/suite/suite.go#L33
+
+### Integration
+
+Integrating the test suite into your implementation can be done using one of
+the following methods:
+
+- The [go test][go-test] command line interface which enables projects of any
+  language to run the test suite on any platform [Golang][go] supports.
+- Using the conformance test suite as a [Golang library][lib] within an already
```

Comment:
`[Golang library][lib]` typo?

Reply:
Similar to above, this just adds a hyperlink which appears to be working? Please un-resolve and let me know if there's still a problem though? :thinking: